util-node/getApiDocsData.js

```diff
@@ -133,6 +133,7 @@ async function getApiDocsData(releaseVersions, callback) {
 
         progressBar.increment();
 
+        if (sortedNavigationEntry.items.length === 0) process.exit(1);
```

Comment:
This isn't really the best way of doing it. I believe it's worth to investigate why the navigation would be empty. This is something that requires research, like going back to trace where and why the entries are not being added/generated.

Reply:
Why it's empty (according to my research): we fetch the data through the GitHub api. And it happens that sometimes we reach the limit/time (you have to read the GitHub docs to be sure). So if it fetches nothing it lists the empty items. That's why I kill the process because if we restart it will loop in the vacuum. You can experiment by syncing 15 times the docs and you'll be that after a while you'll find yourself with your empty items (I had fun syncing the docs)

Reply:
Hmm, I don't think the navigation comes from the GitHub API, and if that was the issue, the Markdown files would be empty, as they're the only content that comes from the API.

The navigation is generated based on the Markdown files. I would suggest you to try to debug the codebase, by running adding some `console.log` statements and see what could be happening. Of course, we don't know what is even triggering this behaviour in these cases.

It's worth to mention this is some complex debugging.

Reply:
I will debugging deeply

Reply:
@AugustinMauroy I can find some time to help you on that if you want